--- thrift-0.9.1/lib/rb/ext/extconf.rb~	2013-08-15 17:04:29.000000000 +0300
+++ thrift-0.9.1/lib/rb/ext/extconf.rb	2013-10-06 02:22:38.662385994 +0300
@@ -24,7 +24,7 @@
 
   $ARCH_FLAGS = Config::CONFIG['CFLAGS'].scan( /(-arch )(\S+)/ ).map{|x,y| x + y + ' ' }.join('')
 
-  $CFLAGS = "-fsigned-char -g -O2 -Wall -Werror " + $ARCH_FLAGS
+  $CFLAGS = "-fsigned-char -g -O2 -Wall " + $ARCH_FLAGS
 
   have_func("strlcpy", "string.h")
 
